lib/vsc/install/shared_setup.py

```diff
@@ -1201,10 +1218,17 @@ def sanitize(name):
             klass = _fvs('sanitize')
             return ",".join([klass.sanitize(r) for r in name])
 
+
+
     @staticmethod
     def get_md5sum(filename):
         """Use this function to compute the md5sum in the KNOWN_LICENSES hash"""
-        return hashlib.md5(open(filename).read()).hexdigest()
+        hasher = hashlib.md5()
+        with open(filename, "rb") as f:
```

Comment:
We try to avoid single-letter variables outsides of list comprehensions, so `f` -> `fh`?